go-controller/pkg/clustermanager/userdefinednetwork/controller/manager/controller.go

```diff
@@ -0,0 +1,44 @@
+package manager
```

Comment:
I tried to avoid adding more to the UDN controller file therefore I seperated the wiring to its own package ( `manager`)
I will be extend to contain the cluster scope controller that will be introduced in a follow up PR.

Reply:
hard to tell how it will look in the future, but for now it has 4 lines of useful code, which looks like not enough for a separate package, maybe you can make it a separate package in the future, wdyt?